selenium_webdriver/src/main/java/core/driver/RemoteChromeDriverManager.java

```diff
@@ -0,0 +1,31 @@
+package core.driver;
+
+import core.enums.DriverType;
+import core.service.TestDataReader;
+import org.apache.commons.lang3.NotImplementedException;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class RemoteChromeDriverManager extends DriverManager {
+
+    private static final String GRID_URL_PROPERTY = "grid.host";
+
+    public RemoteChromeDriverManager() {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setBrowserName(DriverType.CHROME.getBrowser());
+
+        try {
+            driver = new RemoteWebDriver(new URL(TestDataReader.getEnvData(GRID_URL_PROPERTY)), capabilities);
+        } catch (
+                MalformedURLException exception) {
+
+            exception.printStackTrace();
+
+            throw new NotImplementedException("No supported Webdriver were found. " +
```

Comment:
When you face 'MalformedURLException' the problem is in GRID_URL_PROPERTY or Grid setup not in supported Drivers. Please investigate the MalformedURLException cause and fix the thrown Exception (NotImplementedException) and the text

Reply:
Will be fixed during in the next PR.